### PR TITLE
Add basic deploy dashboad to homepage

### PR DESCRIPTION
**Pull request by zachmargolis**
**Why**:
So we can see what's going on

---

It probably needs tests or something

---

Here's a preview:

identity_dashboard 2

### CODE REVIEW

File: app/assets/stylesheets/components/_deploy.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+.deploy-box {
+  min-height: 200px;
+}
+
+.deploy-bg-loading {
+  background: $gray-light;
+}
+
+.deploy-bg-error {
```
Comment (brendansudol) [inline app/assets/stylesheets/components/_deploy.scss]: we already have classes that have the same effect (`bg-red`) -- can you use those?

File: app/assets/stylesheets/components/_deploy.scss [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  background: $red;
+}
+
+.deploy-bg-success {
```
Comment (brendansudol) [inline app/assets/stylesheets/components/_deploy.scss]: `bg-green`

File: app/assets/stylesheets/components/_deploy.scss [hunk 3]
```diff
@@ -0,0 +1,19 @@
+.deploy-hidden {
```
Comment (brendansudol) [inline app/assets/stylesheets/components/_deploy.scss]: please use `display-none` class

File: app/assets/stylesheets/components/_deploy.scss [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  display: none;
+}
+
+.deploy-box {
```
Comment (brendansudol) [inline app/assets/stylesheets/components/_deploy.scss]: not needed

File: app/assets/stylesheets/components/_deploy.scss [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  min-height: 200px;
+}
+
+.deploy-bg-loading {
```
Comment (brendansudol) [inline app/assets/stylesheets/components/_deploy.scss]: can use `bg-gray` or `bg-silver` here

**Comment by brendansudol** [comment]
disregard all of my comments please -- i thought this was in identity-idp; i'm a silly boy

File: config/secrets.yml [hunk 6]
```diff
@@ -1,5 +1,7 @@
 default: &default
   secret_key_base: MGhvYnpIWnJaRHhLaDJrRkpOd1JqWFhHUHMuRFpVeGRhRE5GVHY1NDdBQkwxM0NkWGdnY1NnZmhNMWhCRzBBRG80U2YuTG5hMURrb3NkZ2VURkJOelZkZmhFMUM0TWJwdGFwcFJhM21hRHdQVTFoRUpWZXZCbmp3YnROZkZ0a2MK
+  http_auth_username: <%= ENV["SP_NAME"] %>
+  http_auth_password: <%= ENV["SP_PASS"] %>
```
Comment (pkarman) [inline config/secrets.yml]: yes this is safe to assume.

File: spec/services/deploy_status_checker_spec.rb [hunk 7]
```diff
@@ -0,0 +1,133 @@
+require 'rails_helper'
+
+RSpec.describe DeployStatusChecker do
```
Comment (pkarman) [inline spec/services/deploy_status_checker_spec.rb]: Do we need this `RSpec.` prefix?

Comment (zachmargolis) [inline spec/services/deploy_status_checker_spec.rb]: I'm not sure how we have our project configured, but I like it as it allows us to use RSpec's non-monkeypatching mode where nothing is leaked into the global namespace.

**Pull request merged**